app/presenters/familiar_followers_presenter.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class FamiliarFollowersPresenter
+  class Result < ActiveModelSerializers::Model
+    attributes :id, :accounts
+  end
+
+  def initialize(accounts, current_account_id)
+    @accounts           = accounts
+    @current_account_id = current_account_id
+  end
+
+  def accounts
+    map = Follow.includes(account: :account_stat).where(target_account_id: @accounts.map(&:id)).where(account_id: Follow.where(account_id: @current_account_id).select(:target_account_id)).group_by(&:target_account_id)
+    @accounts.map { |account| Result.new(id: account.id, accounts: account.hide_collections? ? [] : map[account_id].map(&:account)) }
```

Comment:
`Account#hide_collections?` is only useful for remote accounts, use `Account#hides_followers?` instead, which checks both `hide_collections` and the user settings, so works for both local and remote users (though it would require preloading users and their settings to be efficient…)

Reply:
Why are we not using the same boolean for local and remote users? 😕

Reply:
Because one is a user setting that pre-dates the account boolean, and the account boolean was originally meant as two booleans to differentiate between making followers and followed people public (but you thought adding another boolean to accounts would be wasteful)